go/obscuronode/host/host.go

```diff
@@ -34,8 +35,9 @@ type Node struct {
 	stats StatsCollector
 
 	// control the host lifecycle
-	exitNodeCh        chan bool
-	stopNodeInterrupt *int32
+	exitNodeCh          chan bool
+	stopNodeInterrupt   *int32
+	ignoreBootstrapping *int32
```

Comment:
I'd probably call this `boostrapCompleted` or something, else stick a comment next to it to explain what the value of it means

Reply:
Yeah that's a better name -  Updating comments